config_defaults_inc.php

```diff
@@ -3698,14 +3703,15 @@
  *
  * @global integer $g_allow_browser_cache
  */
-# $g_allow_browser_cache = ON;
+$g_allow_browser_cache = OFF;
+
 /**
  * File caching - This will allow the browser to cache downloaded files.
  * Without this set, there may be issues with IE receiving files, and launching
  * support programs.
  * @global integer $g_allow_file_cache
  */
-# $g_allow_file_cache = ON;
+$g_allow_file_cache = OFF;
```

Comment:
This will enable file caching. The reason is that the code that references this config considers it ON if it is set. This is wrong behavior. Hence, I would leave this config and the `$g_allow_browser_cache` disabled until:

- Update the code fetching them to use the standard config_get_global() method.
- Consider them ON only when their value is set to ON.
- Add these two config options to `$g_global_settings` array.

This can be done as part of this PR or as a separate PR.

Reply:
Thanks for getting back to me @vboctor and the feedback.

Gotcha - good to know.
I don't have the time right now to learn where the changes need to go to make this happen :(.
I've altered the commit (and done a force push), to revert the commit explaining why they are not consistent like the others.